pkgs/top-level/python-packages.nix

```
@@ -5008,6 +5008,11 @@ in {
 
   transitions = callPackage ../development/python-modules/transitions { };
 
+  essentia = (toPythonModule (pkgs.essentia.override {
+    pythonPackages = python;
```

Did you mean:

```suggestion
    pythonPackages = self;
```

... here?

---

Also, since the package set is for a particular version, you'd also need to pass `python` here as well.

---

Please tell me what do you think of the last changes I pushed.